jest.config.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
       statements: 100,
     },
   },
+  moduleNameMapper: {'\\.css$': 'identity-obj-proxy'},
```

Comment:
Note: jest was failing because the ChatComposer component imports a CSS file. This is the recommended way to mock CSS files: https://jestjs.io/docs/webpack#mocking-css-modules

I did have to add `identity-obj-proxy` as a new dependency.